.coveragerc

```
@@ -0,0 +1,5 @@
+[run]
+branch = True
+include = wagtail_footnotes/*
+omit = */migrations/*,*/tests/*
```

```suggestion
omit = */migrations/*,*/test/*
```

since all tests are under the `test` dir